_includes/manuals/1.19/3.1.3_puppet_versions.md

```diff
@@ -10,19 +10,12 @@ Foreman 1.18 is the last version that will support Puppet < 4.0. Report/fact pro
     <th><a href="/manuals/{{page.version}}/index.html#3.5.5FactsandtheENC">External node classifier</a></th>
   </tr>
   <tr>
-    <td>0.x-3.7.x</td>
+    <td>0.x-3.x</td>
+    <td>Not supported</td>
     <td>Not supported</td>
-    <td>Untested</td>
```

Comment:
Technically it'll probably still work as no code has been removed so I'd keep this on Untested.